src/register-org/components/document-exchange-reference/document-exchange-reference.component.html

```diff
@@ -7,14 +7,12 @@
       <span class="govuk-caption-xl">Register an organisation</span>
       <form [formGroup]="dxFormGroup">
         <div class="govuk-form-group">
-          <fieldset class="govuk-fieldset" aria-describedby="changed-name-hint">
-            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-              <h1 class="govuk-fieldset__heading">
-                Do you have a document exchange reference for your main office?
-              </h1>
+          <fieldset class="govuk-fieldset">
```

Comment:
@johnbenjamin-hmcts - for accessibility, add `aria-describedby="document-exchange-hint"`

Reply:
Thank you @Daniel-Lam - I have updated the code now.